Makefile

```diff
@@ -179,11 +181,17 @@ endif
 # Generate bundle manifests and metadata, then validate generated files.
 .PHONY: bundle
 bundle: manifests kustomize
+ifneq ($(and $(REPLACES),$(OPERATOR_NAMESPACE)),)
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
+	cd config/manifests/bases && sed -e "s/@REPLACES@/$(REPLACES)/" -e "s/@VERSION@/$(VERSION)/" kustomization-base.yaml > kustomization.yaml
+	cd config/manifests && $(KUSTOMIZE) edit set namespace $(OPERATOR_NAMESPACE)
 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --extra-service-accounts prometheus-k8s --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS) $(BUNDLE_FLAGS)
 	cp config/metadata/* $(OUTPUT_DIR)/metadata/
 	operator-sdk bundle validate $(OUTPUT_DIR)
+else
+	@echo Variables 'REPLACES' and 'OPERATOR_NAMESPACE' are mandatory for generating bundle
+endif
```

Comment:
You can skip this check as we the above requests will guarantee default values for both